app/src/main/java/org/glucosio/android/activity/AddReadingActivity.java

```diff
@@ -134,6 +134,7 @@ public void run() {
                 AnimationTools.startCircularReveal(doneFAB);
             }
         };
+        doneFAB.postDelayed(fabAnimationRunnable, 600);
```

Comment:
There are already `doneFAB.removeCallbacks(fabAnimationRunnable);` exist onDestroy().
Should I do more things?